emoji-substitution/manifest.json

```diff
@@ -0,0 +1,24 @@
+{
+  "manifest_version": 2,
+  "applications": {
+    "gecko": {
+      "strict_min_version": "49.0.1"
```

Comment:
The `strict_min_version` could probably be lower than this, but I didn't test on anything less than `49.0.1`.

It does have a hard limit at `47`, as that's when [let](http://caniuse.com/#search=let) and [const](http://caniuse.com/#search=const) were added.